config.toml

```diff
@@ -106,7 +106,7 @@ weight = 20
 [[menu.reference]]
 parent = "specs"
 name = "Roadmap to 1.0.0"
-url = "https://github.com/ipfs/ipfs/blob/master/ROADMAP-TO-1.0.0.md"
+url = "https://github.com/ipfs/ipfs/blob/master/REQUIREMENTS-TO-1.0.0.md"
```

Comment:
```suggestion
url = "https://github.com/ipfs/ipfs/blob/master/REQUIREMENTS.md"
```

Reply:
This is the right place to link to for now, but it's worth noting that there is a significantly updated version of the IPFS Roadmap in progress that is due to land soon. I think we probably want to rename this link item "Roadmap" and drop the 1.0.0 part, as the roadmap is focused on features and usecases we want to support rather than semver increments. /cc @daviddias

Reply:
Sure, I can see how this may be more useful, thanks for your help @olizilla. I commited your suggestion to this pull request.